docs/content/deprecated/Dropdown.md

```diff
@@ -3,6 +3,12 @@ title: Dropdown
 status: Deprecated
 ---
 
+## Deprecation
+
+Use [DropdownMenu](/DropdownMenu) instead.
```

Comment:
👍 . Just for clarity, can we add the import statement to the top of `DropdownMenu` page? It could be a bit confusing for first-time readers otherwise.

Reply:
Yeah, the DropdownMenu page is a kind of pretty bare 😅